### PR TITLE
spelling: added detailed message to spelling test script

### PR DESCRIPTION
**Pull request by SteLeo1602**
Added additional details to the message to clarify for the beginners what they need to do exactly and ensure they won't confuse the command with the rest of the report.

### CODE REVIEW

**Comment by romani** [comment]
>[ERROR] [checkstyle] [ERROR] C:\projects\checkstyle\.ci\test-spelling-unknown-words.sh:58: Line should not be longer than 100 symbols [lineLength]

Please fix CI failure

File: .ci/test-spelling-unknown-words.sh [hunk 1]
```diff
@@ -55,7 +55,8 @@ echo "Run w"
 
 printDetails() {
   echo ''
-  echo 'If you are ok with the output of this run, you will need to'
+  echo 'If you are ok with the output of this run, you will need \
+to run the following entire multiline command:'
```
Comment (romani) [inline .ci/test-spelling-unknown-words.sh:59]: I would do this in two separate echo commands, no reason on line wrap here.

Comment (romani) [reply .ci/test-spelling-unknown-words.sh:59]: `echo 'If you are ok with the output of this run,'`
Other text by separate echo command.

Comment (SteLeo1602) [reply .ci/test-spelling-unknown-words.sh]: Done

**Pull request merged**